src/Authentication/Model/PaginationSizeOption.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Altinn.Platform.Authentication.Model;
+
+/// <summary>
+/// The pagination size option
+/// </summary>
+public class PaginationSizeOption
```

Comment:
Can we name this bit more generic like PaginationOption? In case we need that to customize something later

Reply:
good idea, 
like turning on or off direct link to pages
will change in new commit

Reply:
Also moved the Option class to the configuration folder, and added Appsettings configuration reading